src/palace/manager/sqlalchemy/model/work.py

```diff
@@ -1662,21 +1668,19 @@ def _set_value(parent, key, target):
         result["licensepools"] = []
         if doc.license_pools:
             for license_pool in doc.license_pools:
-                if not (
-                    license_pool.open_access
-                    or license_pool.unlimited_access
-                    or license_pool.licenses_owned > 0
-                ):
+                if not license_pool.active_status:
                     continue
 
                 lc: dict = {}
                 _set_value(license_pool, "licensepools", lc)
                 # lc["availability_time"] = getattr(item, "availability_time").timestamp()
-                lc["available"] = (
-                    license_pool.unlimited_access or license_pool.licenses_available > 0
+                lc["available"] = license_pool.unlimited_type or (
```

Comment:
Minor: Not part of this PR, but can we might as well get rid of the commented code just above this change.

Reply:
Done